solidserver/data_source_ip_address.go

```diff
@@ -0,0 +1,146 @@
+package solidserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math"
+	"net/url"
+	"regexp"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceipaddress() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceipaddressRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
```

Comment:
There is no need to create a field for the ID, there is one by default that can be set with "d.SetId(<ID>)"

Reply:
assumed that's the input parameter, we should declare it ?

Reply:
I will run some test to validate this point once merged in the branch. Thanks for the clarification.